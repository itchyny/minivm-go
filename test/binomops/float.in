print 3.14 * 6 * 6 - 3 * 4 / 5
