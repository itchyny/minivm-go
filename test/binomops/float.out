111.03999999999999
